src/bindings/js.ts

```diff
@@ -1005,18 +1006,25 @@ export class JSBuilder extends ExportsWalker {
       }
       sb.push(")");
     } else {
-      sb.push(name);
-      if (type.isUnsignedIntegerValue && type.size == 32) {
-        sb.push(" >>> 0");
-      } else if (type == Type.bool) {
-        sb.push(" != 0");
+      // Lift basic plain types
+      if (type.isUnsignedIntegerValue && type.size == 64) {
+        sb.push(`BigInt.asUintN(64, ${name})`);
+      } else {
+        if (type == Type.bool) {
+          sb.push(`${name} != 0`);
+        } else if (type.isUnsignedIntegerValue) {
+          sb.push(`${name} >>> 0`);
+        } else {
+          sb.push(name);
+        }
```

Comment:
Can this perhaps be simplified?
```suggestion
      if (type == Type.bool) {
        sb.push(`${name} != 0`);
      } else if (type.isUnsignedIntegerValue && type.size >= 32) {
        sb.push(type.size == 64 ? `BigInt.asUintN(64, ${name})` : `${name} >>> 0`);
      } else {
        sb.push(name);
```
Then also doesn't append `>>>` on `u8`/`u16`, which can't overflow, as before.